### PR TITLE
feat(datepickerrange): date picker range with new user stories

### PR DESCRIPTION
**Pull request by gayankumara**
… branch

style: added month title for next previous

fix: lint errors fixed

fix: if user click from date if exists then to date also change as from date

fix: if user click on from date after or previous current month then to date picker display correct

feat: if user click from date grater than to on from date picker

feat: if user select from date when already selected then never goes to the to date picker

### CODE REVIEW

**Comment by gayankumara** [comment]
Reopened since Travis CI fails....

**Comment by StommePoes** [comment]
I simply re-ran Travis again. It seems to have passed. All the errors we see in the logs of the failed build are also in v0.

File: docs/src/app/components/containers/components/DatePickerRange.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+      position: 'absolute',
+      backgroundColor:'white',
+      marginTop:'5px'
+    };
+    const divStyle1 = {
+      display: 'inline-block',
+      paddingRight:'5px'
+    };
+
+    return <div>
```
Comment (wyseguyonline) [inline docs/src/app/components/containers/components/DatePickerRange.js]: Incredibly minor nitpick, but the returned HTML should be in parenthesis.

Comment (StommePoes) [inline docs/src/app/components/containers/components/DatePickerRange.js]: Ah yeah, in fact I thought they were required for anything multiline... guess not.

**Comment by wyseguyonline** [comment]
I'm not seeing any issues.

**Comment by JasonLantz** [comment]
i don't see any either...

just to clarify though....

return() in the react render method lets you use multiple lines without the first line of jsx on the same line as the return.

ie. return(
           <div>
                <p>hi there</p>
          </div>
     )

if there's only one line of jsx you can write

return <input blah="blah">

**Comment by JasonLantz** [comment]
screen shot 2016-08-18 at 2 46 08 pm

**Comment by StommePoes** [comment]
@jasonlantz apparently React is okay with not using the () as Gayan had had no errors...

anyway, agreed the only problems I see are outside this particular PR.

There is a larger issue with Isura's related PR where datepicker and datepickerrange are added to index.js, but we'll need to track that down. I think this particular PR can be merged though.

**Pull request merged**